tests/Feature/SubpageTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Subpage;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+#use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class SubpageTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    /**
+     * "Refresh" the database by deleting everything.
+     *
+     * TODO: Use RefreshDatabase instead of DatabaseTransaction/this method.
+     *
+     * @return void
+     */
+    protected function setUp() : void
+    {
+        parent::setUp();
+        DB::table('subpages')->delete();
```

Comment:
I think if we use `RefreshDatabase` instead of `DatabaseTransactions` we don't need to run this? `DB::table('subpages')->delete();`

Reply:
A couple things prevent us from using RefreshDatabase at this time:

1. Our migrations don't yet support `php artisan migrate:fresh`. I added issue #112 to address this.
2. Many of the tests are still dependent on a "real" database dataset. I added issue #113 to address this.